convert2rhel/special_cases.py

```diff
@@ -0,0 +1,29 @@
+import logging
+
+from convert2rhel.systeminfo import system_info
+from convert2rhel.utils import mkdir_p
+
+
+OPENJDK_RPM_STATE_DIR = "/var/lib/rpm-state/"
+
+logger = logging.getLogger(__name__)
```

Comment:
@bocekm there was something about getting the logger in the top-level that had issues. Was it related to importing the file multiple times or something? I don't quite remember

Reply:
It shouldn't be a problem: https://issues.redhat.com/browse/OAMG-3365. Artem can leave it like this. When we have more time, we can change it in every file like that.